Backend/NoCode.FlowerShop.Application/Common/Interfaces/Persistence/IFlowerArrangementCategoryRepository.cs

```diff
@@ -0,0 +1,11 @@
+﻿using NoCode.FlowerShop.Domain;
+
+namespace NoCode.FlowerShop.Application.Common.Interfaces.Persistence;
+
+public interface IFlowerArrangementCategoryRepository
+{
+    Task AddAsync(FlowerArrangementCategory flowerArrangementCategory, CancellationToken token = default);
+    Task<FlowerArrangementCategory?> GetById(Guid id, CancellationToken token = default);
+    Task<List<FlowerArrangementCategory>> GetAllAsync(CancellationToken token = default);
```

Comment:
Not sure that we should return `List` instead `IEnumerable` or `IList` at least. But I guess let it be. Also it might be specific pattern for Entity Framework to return `List` for collections

Reply:
I will make it [IQueryable](https://learn.microsoft.com/en-us/dotnet/api/system.linq.iqueryable?view=net-7.0).

Reply:
Oh yes, should make even more sense